.github/workflows/validate-pr.yml

```diff
@@ -38,10 +38,8 @@ jobs:
       use-scanner-tarballs: true
       v4-tarball-suffix: 'dev-4'
       v5-tarball-suffix: 'dev'
-    secrets: inherit
```

Comment:
I recommend leaving this in. I'm not positive it's necessary here, but I think it might be.